x-pack/legacy/plugins/siem/public/components/header_page/index.tsx

```diff
@@ -91,7 +92,11 @@ const HeaderPageComponent: React.FC<HeaderPageProps> = ({
       <FlexItem>
         {backOptions && (
           <LinkBack>
-            <LinkIcon href={backOptions.href} iconType="arrowLeft">
+            <LinkIcon
+              dataTestSubj={backOptions.dataTestSubj}
```

Comment:
Maybe we should have a default if dataTestSubj is not filled. What do you think?

Reply:
I checked the definition of EUI components and `data-test-subjet` is defined as 

```
export interface CommonProps {
	    className?: string;
	    'aria-label'?: string;
	    'data-test-subj'?: string;
	}
```

There is no need for a default. My mistake.